src/services/analytics/events/overview.ts

```diff
@@ -68,4 +68,12 @@ export const OVERVIEW_EVENTS = {
     action: 'Open Safe Claiming App from widget',
     category: OVERVIEW_CATEGORY,
   },
+  MISSING_SIGNATURES: {
+    action: 'Go to queue from missing signatures',
+    category: OVERVIEW_CATEGORY,
+  },
+  QUEUED_TRANSACTIONS: {
+    action: 'Go to queue from number of queued transactions',
```

Comment:
```suggestion
    action: 'Go to queue from number of queued transactions in sidebar',
```

Reply:
Action != description, it should be a one-two word name, like an id, easily filterable in GA and not subject to edits in future.